### DISCUSSION TITLE
gtkmm comes without schemas

### DISCUSSION THREAD

**Discussion by PointClouder** [microsoft/vcpkg#19058] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I want to build a gtkmm3 application with and open-file-dialog it fails because of missing schemas files. If there is support, I failed to find a documentation

**Proposed solution**
povide a sample schema to use with an application and ideally give it a credible win10 look.

**Describe alternatives you've considered**
Followed this advice:
https://www.gtk.org/download/windows.php
Hardly managed to get it working.

**Comment by Illation** [comment]
I also had this issue and solved it by getting the schema files from this repo as a submodule: https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer
